server/src/test/java/org/opensearch/index/mapper/DynamicTemplatesTests.java

```diff
@@ -131,7 +131,7 @@ public void testSimple() throws Exception {
 
     public void testSimpleWithXContentTraverse() throws Exception {
         String mapping = copyToStringFromClasspath("/org/opensearch/index/mapper/dynamictemplate/simple/test-mapping.json");
-        MapperService mapperService = createMapperService("person", mapping);
```

Comment:
That should also work with `person`, right?

Reply:
No; we're in a transition state right now removing type support. I haven't fully removed types from the MapperService yet. That's coming in a followup PR.

Reply:
👍